### PR TITLE
[patch] Codeship Pro migration

### PR DESCRIPTION
**Pull request by kkortchinska**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Files
+COPY .eslintrc.js .
+COPY global-config.js .
+COPY react-config.js .
+COPY ts-config.js .
+COPY index.js .
+
+# Frequently changed files/folders
+COPY .git ./.git
```
Comment (omgaz) [inline Dockerfile:34]: Do we need to do this, can we just use `COPY .git` (could be wrong) or even `COPY .git .`

Comment (kkortchinska) [reply Dockerfile:34]: I'm afraid we do need it. When you copy a folder over you need to explicitly specify the path. `.git .` will copy over the content to the working directory.

**Comment by simonhoye** [comment]
Could you please list the variables in the 2 encrypted files in the description just so we can see what’s in there? Not the values just the names :)

**Comment by kkortchinska** [comment]
@simonhoye sure! 
GITHUB_OAUTH_TOKEN in both .args and .env
NPM_TOKEN_AHM in both .args and .env
PRIVATE_SSH_KEY in .env

**Comment by simonhoye** [comment]
Why are the GITHUB_OAUTH_TOKEN and NPM_TOKEN_AHM tokens in both?

**Comment by kkortchinska** [comment]
@simonhoye good question! I copied them over from the urls over a month ago while migrating logic and it seems it has been like this since... a very long time? Forever?

**Pull request merged**